src/PlatformCompat.Scanner.Tests/ExceptionScannerTests.cs

```diff
@@ -99,7 +99,7 @@ public C()
             ";
 
             var docIds = @"
-                M:C.#ctor
+                M:C.#ctor @ 0 @ M:C.#ctor
```

Comment:
The `@` is problematic because it can occur in [documentation IDs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/processing-the-xml-file) as well. I suggest we just use whitespace. It's easier to read and illegal for both the number as well as the IDs.